### PR TITLE
Update Vulkan docset to version 1.0.39.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

**Comment by antiagainst** [comment]
@Kapeli: It seems that the connection to GitHub experienced problems on Travis. Would you mind to restart the check?

File: docsets/Vulkan/docset.json [hunk 1]
```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
```
Comment (Kapeli) [inline docsets/Vulkan/docset.json]: You need to add the archive for version `1.0.39`.

File: docsets/Vulkan/docset.json [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
-    "aliases": ["vk", "GLNext", "Graphics API"]
+    "aliases": ["vk", "GLNext", "Graphics API"],
+    "specific_versions": [
+        {
+            "version": "1.0.39",
+            "archive": "versions/1.0.39/Vulkan.tgz"
+        },
+        {
+            "version": "1.0.38",
+            "archive": "versions/1.0.38/Vulkan.tgz"
```
Comment (Kapeli) [inline docsets/Vulkan/docset.json]: You need to add the archive for this version too. Moving the `txt` files I leave behind doesn't work.

File: docsets/Vulkan/docset.json (hunk 1 above)
Comment (antiagainst) [inline docsets/Vulkan/docset.json]: Sorry I don't quite understand. `docsets/Vulkan/versions/1.0.39/Vulkan.tgz` is already there.

Comment (Kapeli) [inline docsets/Vulkan/docset.json]: That's the archive for the `specific_version`. You need to have it at `docsets/Vulkan/Vulkan.tgz` too for the main docset.

File: docsets/Vulkan/docset.json (hunk 2 above)
Comment (antiagainst) [inline docsets/Vulkan/docset.json]: You mean the original `Vulkan.tgz` for `1.0.38`?

Comment (Kapeli) [inline docsets/Vulkan/docset.json]: Yes.

Comment (antiagainst) [inline docsets/Vulkan/docset.json]: Well, it is already removed from the repo and now I'm adding it back again. Sounds a little bit strange, especially we have a new version now and the old version won't be used.

And just to clarify, in the future, say if I want to update 1.0.100, I need to check in all the tarballs for 1.0.38 way up to 1.0.100?

File: docsets/Vulkan/docset.json (hunk 1 above)
Comment (antiagainst) [inline docsets/Vulkan/docset.json]: Okey, I see. Two copies essentially. Will do.

File: docsets/Vulkan/docset.json (hunk 2 above)
Comment (antiagainst) [inline docsets/Vulkan/docset.json]: What probably beneficial is a readme/wiki page describing the procedure of updating docsets. :)

Comment (Kapeli) [inline docsets/Vulkan/docset.json]: > And just to clarify, in the future, say if I want to update 1.0.100, I need to check in all the tarballs for 1.0.38 way up to 1.0.100?

Nope, no need for that. You just need to give the tarball once to the `specific_version`. Think of the stuff in `specific_versions` as completely separate from the main docset archive (i.e. the one at `docsets/Vulkan/Vulkan.tgz`).

Comment (antiagainst) [inline docsets/Vulkan/docset.json]: Okey, I see now.

**Comment by antiagainst** [comment]
@Kapeli: comments addressed. Let me know if further tweaks are needed. :)

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**